WORKSPACE.bzlmod

```diff
@@ -0,0 +1,7 @@
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@rules_android//android:rules.bzl", "android_sdk_repository")
+
+maybe(
```

Comment:
Mirroring what's being done in `rules_android` https://github.com/bazelbuild/rules_android/blob/main/WORKSPACE.bzlmod#L8

Reply:
This will only get called when building within the `rules_kotlin` project. This repo shouldn't be initialized when depending on `rules_kotlin` as a set of rules.